src/avatar-selector.js

```diff
@@ -26,22 +25,16 @@ addLocaleData([...en]);
 
 registerTelemetry();
 
+const hash = new URLSearchParams(location.hash.replace(/^#/, "?"));
 window.APP = new App();
-const hash = queryString.parse(location.hash);
-const isMobile = AFRAME.utils.device.isMobile();
-if (hash.quality) {
-  window.APP.quality = hash.quality;
-} else {
-  window.APP.quality = isMobile ? "low" : "high";
-}
+window.APP.quality = hash.get("quality") || AFRAME.utils.device.isMobile() ? "low" : "high";
```

Comment:
thought: could/should there also be a `const qs = new URLSearchParams(location.search);` so users link to hubs URLs with `&quality=high` in the query-string without having to mess with the hash?

Reply:
I am not sure why this got put into the hash, actually -- I was just mechanically transforming it. It's a good question, though. I will poke around and figure out whether there is some reason for it not to be in the query string.